ratelimits/bucket_test.go

```diff
@@ -14,3 +18,173 @@ func TestNewTransactionBuilder_WithBadLimitsPath(t *testing.T) {
 	_, err = NewTransactionBuilder("testdata/defaults.yml", "testdata/does-not-exist.yml")
 	test.AssertError(t, err, "should error")
 }
+
+func sortTransactions(txns []Transaction) []Transaction {
+	sort.Slice(txns, func(i, j int) bool {
+		return txns[i].bucketKey < txns[j].bucketKey
+	})
+	return txns
+}
+
+func TestNewRegistrationsPerIPAddressTransactions(t *testing.T) {
+	t.Parallel()
+
+	tb, err := NewTransactionBuilder("../test/config-next/wfe2-ratelimit-defaults.yml", "")
+	test.AssertNotError(t, err, "should not error")
```

Comment:
nit: I prefer these to be more descriptive, like
```suggestion
	test.AssertNotError(t, err, "creating TransactionBuilder")
```
so the resulting test failure (if/when it does occur) looks like `test failure: creating TransactionBuilder: parsing default limit: invalid burst` rather than `test failure: should not error: parsing default limit: invalid burst`